lib/gooddata/lcm/actions/ensure_technical_users_domain.rb

```diff
@@ -36,7 +39,7 @@ def call(params)
           domain_name = params.organization || params.domain
           domain = client.domain(domain_name) || fail("Invalid domain name specified - #{domain_name}")
 
-          technical_users = (params.technical_user || []).uniq
+          technical_users = (params.technical_users || params.technical_user || []).uniq
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)